pandas/tests/extension/integer/test_integer.py

```diff
@@ -770,7 +770,7 @@ def test_groupby_mean_included():
     df = pd.DataFrame({
         "A": ['a', 'b', 'b'],
         "B": [1, None, 3],
-        "C": IntegerArray([1, None, 3], dtype='Int64'),
```

Comment:
I would do this in a separate PR

Reply:
Keeping it here, now that we know that the doc PR isn't related.